public/styles/octicons/octicons.scss

```diff
@@ -1,207 +1,209 @@
 @font-face {
```

Comment:
This file is not ours, can we ignore him?

Reply:
If it's not ours, does it make sense to move to `/vendor` folder and `.stylelintignore` that folder?

Reply:
What about using the per-file ignore: https://stylelint.io/user-guide/ignore-code#within-files? I not really would move that file to somewhere other location, since not sure what could go wrong.

Reply:
I'd be okay with doing that for this PR! I think ultimately moving the file to the `/vendor` folder to mark it as `third-party` would make the most sense.

Reply:
He is not fully `third-party`, he just is generated some very old-time ago in #2036, and all part of the code what do the generating removed in #2577, it's one of the things why I wouldn't move this file to `/vendor`. He is is our but not fully ours.

Reply:
Ooh, thanks for the clarification.